setup.py

```diff
@@ -1,52 +1,51 @@
 #!/usr/bin/env python2.6
+#pylint: disable=attribute-defined-outside-init
```

Comment:
I would really suggest NOT telling pylint to ignore things. You can ignore pylint, that's fine, but defining things inside __init__ is good practice. If we didn't think it was, we would have set this to globally ignore such things.